tabs/configuration.js

```diff
@@ -456,6 +456,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             serialRXtypes.push('Spektrum Bidir SRXL');
         }
 
+        if (semver.gte(CONFIG.flightControllerVersion, "3.2.0"))  {
```

Comment:
flightConrollerVersion should not be used, since it breaks compatibility with cleanflight. Please fix this by using apiVersion.